platforms/documentation/docs/src/snippets/daemon-jvm/customized-task/kotlin/build.gradle.kts

```diff
@@ -1,5 +1,30 @@
 // tag::customized-daemon-jvm-task[]
-tasks.updateDaemonJvm {
+tasks.named<UpdateDaemonJvm>("updateDaemonJvm") {
     jvmVersion = JavaLanguageVersion.of(17)
 }
 // end::customized-daemon-jvm-task[]
+
+// tag::customized-plat-daemon-jvm-task[]
+tasks.named<UpdateDaemonJvm>("updateDaemonJvm") {
+    val myPlatform = BuildPlatformFactory.of(
+        org.gradle.platform.Architecture.AARCH64,
+        org.gradle.platform.OperatingSystem.MAC_OS
+    )
+    toolchainPlatforms.add(myPlatform)
+}
+// end::customized-plat-daemon-jvm-task[]
+
+// tag::customized-noplat-daemon-jvm-task[]
+tasks.named<UpdateDaemonJvm>("updateDaemonJvm") {
+    toolchainDownloadUrls.empty()
+}
+// end::customized-noplat-daemon-jvm-task[]
+
+// tag::customized-uri-daemon-jvm-task[]
+tasks.named<UpdateDaemonJvm>("updateDaemonJvm") {
+    toolchainDownloadUrls = mapOf(
+        BuildPlatformFactory.of(org.gradle.platform.Architecture.AARCH64, org.gradle.platform.OperatingSystem.MAC_OS) to uri("https://server?platform=MAC_OS.AARCH64"),
```

Comment:
🤔 Because this requires full package name, we should add a note in the docs about it.
It comes from a conflict on those type names existing in more than one package.
And the other type is first in package alphabetical order.

Reply:
Fixed.